transformer_engine/common/include/transformer_engine/fused_attn.h

```diff
@@ -0,0 +1,188 @@
+/*************************************************************************
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See LICENSE for license information.
+ ************************************************************************/
+
+#ifndef TRANSFORMER_ENGINE_FUSED_ATTN_FP8_H_
+#define TRANSFORMER_ENGINE_FUSED_ATTN_FP8_H_
+
+#include "transformer_engine.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*! \brief Compute dot product attention with packed QKV input.
+ *
+ * Computes: BMM1 -> ScaleMaskSoftmax -> Dropout -> BMM2
+ *
+ *  \param[in]     QKV                   The QKV tensor in packed format,
+ *                                       [total_seqs, 3, head, head_dim].
+ *  \param[in]     Bias                  The B tensor.
+ *  \param[in,out] S                     The S tensor, S = Softmax(P), P = Q * K.T.
+ *  \param[out]    O                     The output O tensor.
+ *  \param[out]    Aux_Output_tensors    Auxiliary output tensors when training, e.g. M, ZInv.
+ *  \param[in]     cu_seqlens            Accumulative sequence lengths, [batch_size + 1].
+ *  \param[in]     rng_state             Seed and offset of CUDA random number generator.
+ *  \param[in]     max_seqlen            Max sequence length used for computing,
+ *                                       it may be >= max(cu_seqlens). 
+ *  \param[in]     is_training           Whether this is in training mode or inference.
+ *  \param[in]     attn_scale            Scaling factor for Q * K.T.
+ *  \param[in]     dropout               Dropout probability.
```

Comment:
It is called p_dropout in the function parameters.

Reply:
Fixed it. It's all dropout now.